pkg/operator/hive/hive_controller.go

```diff
@@ -193,6 +194,30 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Monitor CRDs so that we can keep latest list of supported contracts
+	err = c.Watch(&source.Kind{Type: &apiextv1beta1.CustomResourceDefinition{}},
```

Comment:
It looks like we're missing RBAC here, I don't see anything preexisting in operator_role.yaml. I'm curious how e2e passed, I can't see any errors related to that in the hive-operator pod logs.